presto-main/src/test/java/io/prestosql/sql/planner/assertions/ExpressionVerifier.java

```diff
@@ -409,27 +403,11 @@ protected Boolean visitFunctionCall(FunctionCall actual, Node expectedExpression
 
         FunctionCall expected = (FunctionCall) expectedExpression;
 
-        if (actual.isDistinct() != expected.isDistinct()) {
-            return false;
-        }
-
-        if (!actual.getName().equals(expected.getName())) {
-            return false;
-        }
-
-        if (!process(actual.getArguments(), expected.getArguments())) {
-            return false;
-        }
-
-        if (!process(actual.getFilter(), expected.getFilter())) {
-            return false;
-        }
-
-        if (!process(actual.getWindow(), expected.getWindow())) {
-            return false;
-        }
-
-        return true;
```

Comment:
it was written that way to ease debugging

Reply:
How did that work?

Reply:
I used to set a breakpoint at `return` so it was obvious which branch was used. It is not a big deal though. Nevermind.

Reply:
I understand. Hmm.. but we don't use series of `if`s in other cases, so .... this commit makes the code in the class more consistent.
I'm OK if you change other `&&` to separate `if`s though.

Reply:
It is been a while since I last debugged this code. Hope not to get back to it soon. Please, nevermind.